src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -97,21 +110,88 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
     toggle: togglePersonalisedSchedule,
   } = useShowHide(false);
 
+  const liveAndFutureEvents = useMemo(
+    () =>
+      relatedVenueEvents.filter(
+        (event) =>
+          secondsToMilliseconds(
+            event.start_utc_seconds + minutesToSeconds(event.duration_minutes)
+          ) > startOfToday().getTime()
+      ),
```

Comment:
can't we just use `relatedVenueEvents.filter(isEventLiveOrFuture)` ?

Reply:
yep, fixed b41ef61